datamade/la-metro-councilmatic
# Configure Haystack to raise exceptions, default to batch size of 100

## Overview

See title. 

### Checklist


## Testing Instructions

 * In your copy of `settings_deployment.py`, change the Solr URL in `HAYSTACK_CONNECTIONS` to `http://solr:8982/solr/lametro` (i.e., the wrong URL).
 * Run `update_index` and confirm it fails and the command exits: `docker-compose run --rm app python manage.py update_index --batch-size=1 --max-retries=1 --verbosity=2`.
 * Revert your change to the Solr URL and re-run the above command. Confirm updates succeed. (You can interrupt the update after you confirm.)

Handles #535, supports #538.
